contrib/postgresql/template.py

```diff
@@ -0,0 +1,106 @@
+pkgname = "postgresql"
+pkgver = "16.2"
+pkgrel = 0
+build_style = "configure"
+configure_args = [
+    # TODO: review path choices - alpine puts pkgname in a few of these
```

Comment:
these are all the defaults (and map the same way to meson)- alpine changes them because alpine supports multiple versions

tbh, having a postgresql without packaging multiple versions (read: current, and N-1), to allow upgrading, is useless long term. on any major everyone would break their database because you need the old binaries around to upgrade it. so you actually do need to name this postgresql16 and do some setup to allow a 17 later, which does involve changing these. but we can think about it later